src/app/campaign/store/effects/account.effects.ts

```diff
@@ -1,21 +1,21 @@
 import { Injectable } from '@angular/core';
-import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
 import { of } from 'rxjs';
 import { catchError, withLatestFrom, switchMap, map } from 'rxjs/operators';
 import { UserService } from '../../../core';
 import * as accountActions from '../actions/account-action.creator';
-import { ActionTypes } from '../actions/action.types';
 
 @Injectable()
 export class AccountEffects {
-  @Effect()
-  loadAccounts$ = this.actions$.pipe(
-    ofType(ActionTypes.CAMPAIGN_ACCOUNTS_LOAD),
-    switchMap(() => this.user.accounts),
-    withLatestFrom(this.user.defaultAccount),
-    map(([accounts, defaultAccount]) => new accountActions.AccountsLoadSuccess({ docs: [defaultAccount].concat(accounts) })),
-    catchError(error => of(new accountActions.AccountsLoadFailure({ error })))
```

Comment:
createEffect adds type safety. Fails with compile time error if the effect does not return an Action or have `{dispatch: false}`